static/styles/zulip.css

```diff
@@ -584,6 +584,13 @@ td.pointer {
     transition: background-color 2.7s ease-in, color 2.7s ease-in;
 }
 
+.alert_copy {
+    font-size: 14px;
+    color: green;
+    font-weight: 600;
```

Comment:
Let's make this font weight 400, and also add a few pixels of margin or padding to the right. 
It also seems to be not aligned vertically with the timestamp and the rest of the line?